post_office/apps.py

```diff
@@ -1,17 +1,14 @@
 from django.apps import AppConfig
 from django.utils.translation import ugettext_lazy as _
-from django.utils.module_loading import import_string
 
 
 class PostOfficeConfig(AppConfig):
     name = 'post_office'
     verbose_name = _("Post Office")
 
     def ready(self):
+        from post_office import tasks
         from post_office.signals import email_queued
 
-        try:
-            queued_mail_handler = import_string('post_office.tasks.queued_mail_handler')
```

Comment:
What about leaving `import_string` as it is? If `celery` can't be imported, then this will fail as well and will be catched by the exception handler below.

This is just a question and not a change request.